specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/examples/StorageAccountCreateUserAssignedEncryptionIdentityWithCMK.json

```diff
@@ -0,0 +1,118 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "res9101",
+    "accountName": "sto4445",
+    "api-version": "2020-08-01-preview",
+    "parameters": {
+	   "identity": {
+		 "type": "UserAssigned", 
+		 "userAssignedIdentities": { 
+			 "/subscriptions/{subscription-id}/resourceGroups/res9101/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{managed-identity-name}": {
+		   } 
+	   }
+      },
+      "sku": {
+        "name": "Standard_LRS"
+      },
+      "kind": "Storage",
+      "location": "eastus",
+      "properties": {
+        "encryption": {
+          "services": {
+            "file": {
+              "keyType": "Account",
+              "enabled": true
+            },
+            "blob": {
+              "keyType": "Account",
+              "enabled": true
+            }
+          },
+		  "keyvaultproperties": {
+            "keyvaulturi": "https://myvault8569.vault.azure.net",
+            "keyname": "wrappingKey",
+            "keyversion": ""
+          },
+          "keySource": "Microsoft.Keyvault"
+        }
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "identity": {
+          "userAssignedIdentities": {
+            "/subscriptions/{subscription-id}/resourceGroups/res9101/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{managed-identity-name}": {
+              "principalId": "8d823284-1060-42a5-9ec4-ed3d831e24d7",
+              "tenantId": "f14da12a-1e85-4c31-ae98-bcf5608d2266",
+              "clientId": "fbaa6278-1ecc-415c-819f-6e2058d3acb5"
+            }
+          },
+          "type": "UserAssigned"
+        },
+        "sku": {
+          "name": "Standard_LRS",
+          "tier": "Standard"
+        },
+        "kind": "StorageV2",
+        "id": "/subscriptions/{subscription-id}/resourceGroups/res9101/providers/Microsoft.Storage/storageAccounts/sto4445",
+        "name": "sto4445",
+        "type": "Microsoft.Storage/storageAccounts",
+        "location": "eastus",
+        "tags": {},
+        "properties": {
+          "privateEndpointConnections": [],
+          "networkAcls": {
+            "bypass": "AzureServices",
+            "virtualNetworkRules": [],
+            "ipRules": [],
+            "defaultAction": "Allow"
+        },
+        "supportsHttpsTrafficOnly": true,
+        "encryption": {
+          "identity": {
```

Comment:
Is the identity in Encryption only for output from server? the request seems not set it.
If so , it might should be a read only property in Encryption.
If not, please include sample to set it in request.

Reply:
The request can set this property and the swagger PR has been updated to reflect the same.